frontend/test/playwright/e2e/recent-searches.spec.ts

```diff
@@ -100,53 +133,76 @@ breakpoints.describeMobileXsAndDesktop(({ breakpoint }) => {
     const recentSearchesText = await getRecentSearchesText(page)
     expect(recentSearchesText).toContain(noRecentLabel)
   })
+})
 
-  test("can open recent searches with keyboard", async ({ page }) => {
-    await executeSearches(page)
-    await tabToSearchbar(page)
-    await page.keyboard.press("ArrowDown")
+breakpoints.describeMobileXsAndDesktop(({ breakpoint }) => {
```

Comment:
In a fast-follow PR, we should add a test that ensures the search box can be clicked when the recent searches is open at each breakpoint.